tests/data/test_scatterPlot_lines/line_0.txt

```diff
@@ -1,33 +1,38 @@
-2.165581377360398818e+01 2.537420191914861789e+00
```

Comment:
These tests are really annoying but I think I will make a future ticket to sort out testing in atools in general.

Reply:
Nah, it's great.